seatunnel-translation/seatunnel-translation-base/src/main/java/org/apache/seatunnel/translation/source/CoordinatedSource.java

```diff
@@ -231,6 +244,7 @@ public Map<Integer, List<byte[]>> snapshotState(long checkpointId) throws Except
         StateT enumeratorState = splitEnumerator.snapshotState(checkpointId);
         if (enumeratorState != null) {
             byte[] enumeratorStateBytes = enumeratorStateSerializer.serialize(enumeratorState);
+            // -1 identification that the last status information is read
```

Comment:
Would that be a better way to describe it ？"-1 represents the state of an pending split in SourceSplitEnumerator."